server/config/config.go

```diff
@@ -325,6 +327,8 @@ func (man Manager) addConfigs() {
 	man.addConfigString("pubsub.project", "", "Google Cloud Project to use")
 	man.addConfigString("pubsub.status_topic", "", "PubSub topic for status logs")
 	man.addConfigString("pubsub.result_topic", "", "PubSub topic for result logs")
+	man.addConfigString("pubsub.include_attributes", "", "Comma-separated list of json keys to include as PubSub attributes")
+	man.addConfigString("pubsub.decoration_attributes", "", "Comma-separated list of decorations to include as PubSub attributes")
```

Comment:
these get exposed to GCP pubsub too, just from a different place in the osquery message.  Consider prefixing with `d_` or something so there's no potential conflict with others.

Reply:
I could go either way on this. If we do prefix it I'd prefer to use the longer form `decorator_` so that it's easier to interpret.